xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -173,11 +223,15 @@
         <Parameter Name="escape" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="utf8PropertyName">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="escape">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="utf8PropertyName">The UTF-8 encoded property name of the JSON object to be written.</param>
+        <param name="value">The value to be written as a JSON literal "true" or "false" as part of the name/value pair.</param>
+        <param name="escape"><see langword="false" /> to indicate the writer should assume the property name is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter and its default value is <see langword="true" />.</param>
```

Comment:
```suggestion
        <param name="escape"><see langword="false" /> to indicate the writer should assume the property name is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter; its default value is <see langword="true" />.</param>
```